libs/estdlib/src/lists.erl

```diff
@@ -447,16 +465,12 @@ seq(From, To, Incr) when
 seq(To, To, 0) ->
     [To];
 seq(From, To, Incr) ->
-    seq(From, To, Incr, []).
+    Last = From + ((To - From) div Incr) * Incr,
+    seq_r(From, Last, Incr, []).
 
 %% @private
-seq(From, To, Incr, Accum) when
-    (Incr > 0 andalso From > To) orelse
-        (Incr < 0 andalso To > From)
-->
-    reverse(Accum);
-seq(From, To, Incr, Accum) ->
-    seq(From + Incr, To, Incr, [From | Accum]).
+seq_r(From, From, _Incr, Acc) -> [From | Acc];
```

Comment:
I guess this passes erlfmt?  Okay if so then

Reply:
It does.